nautobot/core/ui/object_detail.py

```diff
@@ -895,6 +899,74 @@ def render_body_content(self, context):
         return result
 
 
+class _BaseTextPanel(Panel):
```

Comment:
Since this has non-private subclasses, I wonder if it should be non-private (`BaseTextPanel`) as well?